packages/ocaml-lua/ocaml-lua.1.8/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "p.donadeo@gmail.com"
+authors: [
+  "Paolo Donadeo"
+  "Sylvain Le Gall"
+]
+homepage: "https://pdonadeo.github.io/ocaml-lua/"
+bug-reports: "https://github.com/pdonadeo/ocaml-lua/issues"
+dev-repo: "git+https://github.com/pdonadeo/ocaml-lua.git"
+license: "MIT"
+build: [
+  [ "sh" "-c" "cd src/lua_c ; tar xf lua-5.1.5.tar.gz" ]
+  [ "sh" "-c" "cd src/lua_c/lua-5.1.5 ; patch -p1 -i ../lua.patch" ]
+  [ "sh" "-c" "cd src/lua_c ; mv lua-5.1.5 lua515" ]
+  [ "dune" "build" "@install" ]
+  [ "dune" "build" "@doc" ] {with-doc}
+]
+install: [ "dune" "install" ]
+remove: [[ "dune" "uninstall" ]]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
Dune shouldn't be tagged as a build dependency (see #14266). Furthermore the `dune-project` file in the source repository says this package requires at least dune 1.11